tiledb/sm/query/sparse_index_reader_base.cc

```diff
@@ -337,9 +388,36 @@ Status SparseIndexReaderBase::compute_tile_bitmaps(
     return Status::Ok();
   }
 
+  // Compute parallelization parameters.
+  uint64_t num_cell_ranges = 1;
```

Comment:
Let's choose a better name here. This is essentially `num_range_threads`, right?